app/posts/claim-funding-for-mentors/2024-10-15-accessibility-audit.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Accessibility audit and remediation
+description: What we've done to ensure the accessibility of our service
+date: 2024-10-15
+tags:
+  - accessibility
```

Comment:
To avoid these strange breadcrumbs:

![Screenshot 2024-11-18 at 17 52 29@2x](https://github.com/user-attachments/assets/ffc94d65-942e-4bbb-92e9-22a65cf1b212)

You can add a unique key:

```suggestion
  - accessibility
eleventyComputed:
  eleventyNavigation:
    key: claim-funding-accessibility-audit
```